arwen/errorUserMsg.go

```diff
@@ -0,0 +1,5 @@
+package arwen
+
+const UserErrorArgIndexOutOfRange = "argument index out of range"
```

Comment:
`bigIntOps` etc. contain some hardcoded error messages. Perhaps you can also move the here? Can be done in another PR, of course.